rllib/algorithms/algorithm.py

```diff
@@ -3215,7 +3216,17 @@ def _run_one_training_iteration(self) -> Tuple[ResultDict, "TrainIterCtx"]:
                             "one single result dict per training iteration."
                         )
 
-        # Only here, reduce the results into a single result dict.
+                    # TODO (sven): Resolve this metric through log_time's future
+                    #  ability to compute throughput.
+                    self.metrics.log_value(
```

Comment:
Great question. For PPO, this is not applicable. Each loss reported is the only loss for that iteration. Or do you mean SGD minibatch losses?? Those are lost for now, since all losses are always logged with `window=1`.